mypyc/test-data/run-classes.test

```diff
@@ -1636,3 +1636,56 @@ from native import A, B, C
 a = A()
 b = B()
 c = C()
+
+[case testAttrOverrides]
+from typing import Any
+
+class Proxy:
+    def __setattr__(self, name:str, value: Any) -> None:
+        print("__setattr__")
```

Comment:
Can you print the arguments here, to make it easier to follow the output? It would be good to do it in the other methods as well.